setup.py

```diff
@@ -50,14 +50,14 @@ def read(fname):
 
     install_requires=[
         'pyarrow>=14.0.0',
-        'numba==0.59.1',
-        'numpy==1.26.4',
-        'tables==3.9.1',
+        'numba>=0.59.1,<=0.61.2',
+        'numpy>=1.26.4,<2.0.0',
+        'tables>=3.9.1',
         'six>=1.16.0',
-        'pandas==2.0.2 ',
-        'pytest==8.3.3',
-        'scipy==1.14.1',
-        'psutil==6.1.0',
+        'pandas>=2.0.2,<2.2.0',
+        'pytest>=8.3.3',
+        'scipy>=1.14.1',
+        'psutil>=6.1.0',
         'fastparquet==2023.4.0' #Will drop support of fastparquet in future versions
```

Comment:
I forget, do we use parquet in GAS still? If so, should this be changed from `==` to `>=`?

Reply:
https://github.com/phac-nml/genomic_address_service/pull/47